File: lowestValue.c
#include <stdio.h>

int main() {

    int array[] = {10, 3, 8, 5, 7};
    int size = sizeof(array) / sizeof(array[0]);
    int min = array[0];

    for (int i = 0; i < size; i++) {
        if (array[i] < min) {
            min = array[i];
        }
    }

    printf("Lowest value in this array is %d.", min);
    return 0;
}

File: lowestValue.java
import java.util.Arrays;

public class lowestValue {
    public static void main(String[] args) {

        int [] array = {19, 34, 6, 22, 199};

        int size =  array.length;
        int min =  array[0];

        for (int i = 0; i < size; i++) {
            if (array[i] < min) {
                min = array[i];
            }
        }

        System.out.println("Lowest value in this array is " + min);
    }
}
